bindinfo/handle.go

```diff
@@ -213,7 +213,9 @@ func (h *BindHandle) CreateBindRecord(sctx sessionctx.Context, record *BindRecor
 		h.bindInfo.Unlock()
 	}()
 
-	txn, err1 := h.sctx.Context.Txn(true)
+	txn, err1 := h.sctx.Context.Txn(true, func(txnOpt *sessionctx.TxnOption) {
```

Comment:
I think we can define this closure as a global variable and use that variable. In most cases, we just need to enable `GlobalTxnScope`, the boilerplate is too annoying.